ajenti-core/aj/gate/stream.py

```diff
@@ -34,9 +35,10 @@ def __init__(self, obj, endpoint):
         self.endpoint = endpoint
 
     def serialize(self):
+        self.object_tmp = {k : (v.decode()  if isinstance(v, six.binary_type) else v) for k, v in self.object.items()}
         return {
-            'id': self.id,
-            'object': self.object,
+            'id': self.id.decode(),
```

Comment:
Let's instead keep self.id a `str` (as it's supposed to be text, not bytes)

Reply:
The problem is that binascii.hexlify(os.urandom(32)) deliver a bytes in PY3. The .decode() method will convert this as str.

Reply:
Then we can just call .decode() right away in __init__ and keep everything uniform